packages/dev/test/PaymasterCommitment.test.ts

```diff
@@ -185,10 +185,10 @@ contract('Paymaster Commitment', function ([_, relayOwner, relayManager, relayWo
       // gasPrice is '1', so price=gasUsed...
       expectEvent(res, 'TransactionRelayed', { status: '0' })
 
-      const gasUsed = res.receipt.gasUsed
+      // const txCost = toBN(res.receipt.gasUsed).mul(toBN(gasPrice))
```

Comment:
why is it commented out?

Reply:
Initially I fixed the gasPrice to be higher than 1 since it's needed post london, then I realized that the charge is being calculated with the client's gasPrice in request, and the relayer can simply give actual gasPrice which is higher. So, gasPrice of 1 removes the need to multiply everywhere as the comment says. 

TLDR; this line should be removed.